src/accessibility/outputs.js

```diff
@@ -0,0 +1,418 @@
+/**
+ * @module Environment
+ * @submodule Environment
+ * @for p5
+ * @requires core
+ */
+
+// Errors when running example:
+// - point
+// - line
+// - appending does not work
+// Call update only at the end of draw?
```

Comment:
seeing your comment here. updating at the end makes sense (otherwise will the reader be notified of changes multiple times?)